project/src/main/java/com/google/autograder/data/Database.java

```diff
@@ -201,18 +205,22 @@ public static void addQuestion(String questionName, String questionType, int que
         save(questionEntity);
     }
 
-    public static void addSubmission(Entity assignmentEntity) {
+    public static void addSubmission(Entity assignmentEntity, String blobKey) {
         Entity submissionEntity = new Entity("Submission");
         submissionEntity.setProperty("graded", "NOT_GRADED");
         submissionEntity.setProperty("assignmentKey", assignmentEntity.getKey());
```

Comment:
should we be storing Key as String? ("KeyFactory.keyToString(assignmentEntity.getKey())")

Reply:
I think it's fine to store a Key (https://cloud.google.com/appengine/docs/standard/java/datastore/entities#Properties_and_value_types)